src/ratesConfig.ts

```diff
@@ -0,0 +1,41 @@
+import Big from "big.js";
+import debug = require("debug");
+import Asset from "./asset";
+const log = debug("bobtimus:rates");
+
+export interface TradeAmounts {
+  buyAsset: Asset;
+  sellAsset: Asset;
+  buyNominalAmount: Big;
+  sellNominalAmount: Big;
+}
+
+export type RatesConfig = {
+  [buyAsset in Asset]: {
+    [sellAsset in Asset]?: {
+      maxSell: number;
+    }
+  }
+};
+
+/// Returns Buy divided by Sell (Sell-Beta in exchange terms) rate based on the configuration
+export function isTradeAcceptable(
```

Comment:
:upside_down_face: I find the logic here hard to follow :confused: Might just be the names chosen.

Reply:
It's maths, you know the maths, maybe proposed new names? Happy to call and explain. I have attempted a rename.

Reply:
I had a chat with @thomaseizinger and we did not come with something better. Maybe let's chat face-to-face :)